packages/@expo/fingerprint/src/utils/Path.ts

```diff
@@ -1,4 +1,4 @@
-import minimatch, { type IMinimatch } from 'minimatch';
+import { Minimatch, MinimatchOptions } from 'minimatch';
```

Comment:
```suggestion
import { Minimatch, type MinimatchOptions } from 'minimatch';
```

Reply:
Ah, totally missed this! We don't do this consistently. Should we spin out a task for enabling `isolatedModules`? Feels like that's the intention in `@expo/cli` and will start affecting more efforts

Reply:
unclear to me. i also have no idea whether we should use `import type { ... } from ...` or `import { type ... } from ...`. up to anyway if you have some recommendation.